website/content/en/docs/main/tutorials/nginx/index.md

```diff
@@ -0,0 +1,341 @@
+
+In this tutorial, we'll establish connectivity across clusters using ClusterLink to access a remote Nginx server.
+The tutorial uses two kind clusters:
+
+1) Client cluster - runs ClusterLink along with a client.
+2) Server cluster - runs ClusterLink along with a Nginx server.
+
+## Install ClusterLink CLI
+
+{{% include "../shared/cli-installation.md" %}}
+
+## Initialize clusters
+
+In this tutorial we set up a local environment using [kind][].
+ You can skip this step if you already have access to existing clusters, just be sure to
+ set KUBECONFIG accordingly.
+
+To setup two kind clusters:
+
+1. Install kind using [kind installation guide][].
+1. Create a directory for all the tutorial files:
+
+    ```sh
+    mkdir nginx-tutorial
+    ```
+
+1. Open two terminals in the tutorial directory and create a kind cluster in each terminal:
+
+    *Client cluster*:
+
+    ```sh
+    cd nginx-tutorial
+    kind create cluster --name=client
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    cd nginx-tutorial
+    kind create cluster --name=server
+    ```
+
+   {{< notice note >}}
+   kind uses the prefix `kind`, so the name of created clusters will be **kind-client** and **kind-server**.
+   {{< /notice >}}
+
+1. Setup `KUBECONFIG` on each terminal to access the cluster:
+
+    *Client cluster*:
+
+    ```sh
+    kubectl config use-context kind-client
+    cp ~/.kube/config $PWD/config-client
+    export KUBECONFIG=$PWD/config-client
+    ```
+
+    *Server cluster*:
+
+    ```sh
+    kubectl config use-context kind-server
+    cp ~/.kube/config $PWD/config-server
+    export KUBECONFIG=$PWD/config-server
+    ```
+
+{{< notice tip >}}
+You can run the tutorial in a single terminal and switch access between the clusters
+using `kubectl config use-context kind-client` and `kubectl config use-context kind-server`.
+{{< /notice >}}
+
+## Deploy nginx client and server
+
+Setup the ```TEST_FILES``` variable, and install nginx on the server cluster.
+
+*Client cluster*:
+
+```sh
+export TEST_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/nginx/testdata
+```
+
+*Server cluster*:
+
+```sh
+export TEST_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/nginx/testdata
+kubectl apply -f $TEST_FILES/nginx-server.yaml
+```
+
+## Deploy ClusterLink
+
+{{% include "../shared/deploy-clusterlink.md" %}}
+
+## Enable cross-cluster access
+
+In this step, we enable connectivity access between the client and server.
+ For each step, you have an example demonstrating how to apply the command from a
+ file or providing the complete custom resource (CR) associated with the command.
+
+Note that the provided YAML configuration files refer to environment variables
+ (defined below) that should be set when running the tutorial. The values are
+ replaced in the YAMLs using `envsubst` utility.
+
+{{% expand summary="Installing `envsubst` on macOS" %}}
+In case `envsubst` does not exist, you can install it with:
```

Comment:
this section (installing envsubst and MacOS) seems to repeat itself between tutorials. 
Can it be included instead of being replicated?